com.microsoft.teams.Teams.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "com.microsoft.teams.Teams",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "19.08",
+  "command": "teams",
+  "tags": [
+    "proprietary"
+  ],
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=all",
+    "--filesystem=home",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.gtk.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "ar",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 /usr/lib/$(gcc -print-multiarch)/libbfd-*.so -t ${FLATPAK_DEST}/lib/",
+        "install -Dm0755 $(which ar) ${FLATPAK_DEST}/bin/"
+      ]
+    },
+    {
+      "name": "desktop-file-edit",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 $(which desktop-file-edit) ${FLATPAK_DEST}/bin/"
+      ]
+    },
+    {
+      "name": "libsecret",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dmanpage=false",
+        "-Dvapi=false",
+        "-Dgtk_doc=false"
+      ],
+      "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/pkgconfig",
+        "/share/gir-1.0",
+        "/share/man",
+        "*.la"
+      ],
+      "sources": [
+        {
+          "commit": "f576dfbe50f9aed22802ad0db0356d081f467d7a",
+          "tag": "0.19.1",
+          "type": "git",
+          "url": "https://gitlab.gnome.org/GNOME/libsecret.git"
+        }
+      ]
+    },
+    {
+      "name": "teams",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 apply_extra.sh ${FLATPAK_DEST}/bin/apply_extra",
+        "install -Dm0755 teams.sh ${FLATPAK_DEST}/bin/teams",
+        "install -Dm0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "apply_extra.sh"
+        },
+        {
+          "type": "file",
+          "path": "teams.sh"
+        },
+        {
+          "type": "file",
+          "path": "com.microsoft.teams.Teams.metainfo.xml"
+        },
+        {
+          "type": "extra-data",
+          "filename": "teams.deb",
+          "only-arches": [
```

Comment:
No need to define it, flathub.json already limits target architectures.

Reply:
It's required by the external data checker.

Reply:
Only for Flash checker.

Reply:
[Nope](https://github.com/flathub/flatpak-external-data-checker/blob/master/src/checkers/debianrepochecker.py#L109).

Reply:
It should be changed to default to x86_64. I may take a look at it today.

Reply:
I'll try to come up with something as well if I found time.

Reply:
Please see https://github.com/flathub/flatpak-external-data-checker/pull/61